src/stringIndices.js

```diff
@@ -0,0 +1,13 @@
+const readlineSync = require('readline-sync')
+
+const collectionOfWords = readlineSync.question('Write a funny sentence\n')
+const newCollection = collectionOfWords.split(' ')
+const index = readlineSync.question(`pick a number between 1 and ${newCollection.length}?\n`)
+
+const stringIndices = ( newCollection, index ) => {
+  return index > newCollection.length || index <= 0
```

Comment:
I spent a while trying to parse this ternary . . . 
It might be just a me-thing, but I'd be better able to understand this if it were an if-statement.

I do love ternaries! Maybe I'm just sleepy this morning . . .